docs/reference/mount.md

```diff
@@ -99,6 +99,10 @@ Flags
 
 ## Examples
 
+**Note**
+⚠️ For simplicity the examples show `main` as the ref, Everest will always mount a specific commit ID, given a ref it will use the HEAD (e.g most recent commit).
```

Comment:
```suggestion
⚠️ For simplicity, the examples show `main` as the ref, Everest will always mount a specific commit ID, given a ref it will use the HEAD (e.g most recent commit).
```